source/includes/fundamentals/code-examples/BulkWrite.cs

```diff
@@ -0,0 +1,201 @@
+        var updateOneModel = new BulkWriteUpdateOneModel<BsonDocument>(
+            "sample_restaurants.restaurants",
+            Builders<BsonDocument>.Filter.Eq("name", "Mongo's Deli"),
+            Builders<BsonDocument>.Update.Set("cuisine", "Sandwiches and Salads")
+        );
+        // end-bulk-update-one
+    }
+
+    static void UpdateMany()
+    {
+        // start-bulk-update-many
+        var updateManyModel = new BulkWriteUpdateManyModel<BsonDocument>(
+            "sample_restaurants.restaurants",
+            Builders<BsonDocument>.Filter.Eq("name", "Mongo's Deli"),
+            Builders<BsonDocument>.Update.Set("cuisine", "Sandwiches and Salads")
+        );
+        // end-bulk-update-many
+    }
+
+    static void ReplaceOne()
+    {
+        // start-bulk-replace-one
+        var replaceOneModel = new BulkWriteReplaceOneModel<BsonDocument>(
+            "sample_restaurants.restaurants",
+            Builders<BsonDocument>.Filter.Eq("restaurant_id", "1234"),
+            new BsonDocument{
+                { "name", "Mongo's Pizza" },
+                { "cuisine", "Pizza" },
+                { "borough", "Brooklyn" },
+                { "restaurant_id", "5678" }
+            }
+        );
+        // end-bulk-replace-one
+    }
+
+    static void DeleteOne()
+    {
+        // start-bulk-delete-one
+        var deleteOneModel = new BulkWriteDeleteOneModel<BsonDocument>(
+            "sample_restaurants.restaurants",
+            Builders<BsonDocument>.Filter.Eq("restaurant_id", "5678")
+        );
+        // end-bulk-delete-one
+    }
+
+    static void DeleteMany()
+    {
+        // start-bulk-delete-many
+        var deleteManyModel = new BulkWriteDeleteManyModel<BsonDocument>(
+            "sample_restaurants.restaurants",
+            Builders<BsonDocument>.Filter.Eq("name", "Mongo's Deli")
+        );
+        // end-bulk-delete-many
+    }
+
+    static void BulkWriteSync()
+    {
+        // start-bulk-write-sync
+        var client = new MongoClient("mongodb://localhost:27017");
+        var collection = "sample_restaurants.restaurants";
+
+        var bulkWriteModels = new WriteModel<BsonDocument>[]
+        {
+            new BulkWriteInsertOneModel<BsonDocument>(
+                collection,
+                new BsonDocument{
+                    { "name", "Mongo's Deli" },
+                    { "cuisine", "Sandwiches" },
+                    { "borough", "Manhattan" },
+                    { "restaurant_id", "1234" }
+                }
+            ),
+            new BulkWriteInsertOneModel<BsonDocument>(
+                collection,
+                new BsonDocument{
+                    { "name", "Mongo's Deli" },
+                    { "cuisine", "Sandwiches" },
+                    { "borough", "Brooklyn" },
+                    { "restaurant_id", "5678" }
+                }
+            ),
+            new BulkWriteUpdateManyModel<BsonDocument>(
+                ,collection
```

Comment:
S: something looks off here
```suggestion
            new BulkWriteUpdateManyModel<BsonDocument>(
                collection,
```